ui/apps/platform/src/Components/ReportJobsTable.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { Bullseye, Button, Text } from '@patternfly/react-core';
+import {
+    ActionsColumn,
+    ExpandableRowContent,
+    Table,
+    Tbody,
+    Td,
+    Th,
+    Thead,
+    Tr,
+} from '@patternfly/react-table';
+
+import useSet from 'hooks/useSet';
+import useAuthStatus from 'hooks/useAuthStatus';
+import { ReportSnapshot } from 'services/ReportsService.types';
+import { saveFile } from 'services/DownloadService';
+import { getDateTime } from 'utils/dateUtils';
+import ReportJobStatus from 'Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/ReportJobStatus';
+import { ComplianceScanSnapshot } from 'services/ComplianceScanConfigurationService';
+import EmptyStateTemplate from './EmptyStateTemplate';
```

Comment:
Relative import at `./…` sibling level brings question to mind about pro and con of Report folder?

Reply:
Would you happen to have any preferences here? We do have a `ReportJobsHelpAction` and a `ReportJobsTable` component. We could add them under a `Reports` folder. Since there are only two for now, I haven't done that. If it's easier to organize mentally with the directory, I can add that too